### ISSUE TITLE
Problema en terminal

### ISSUE THREAD

**Issue by mesaglio** [MumukiProject/mumuki-guia-text-archivos-repositorios-y-versiones#1]
No reconoce ningun comando. ☺️ 
![image](https://user-images.githubusercontent.com/29051129/61011376-b754a600-a350-11e9-9843-86f2d35b0bf3.png)

**Comment by mesaglio** [comment]
Capaz también este relacionado este.
![image](https://user-images.githubusercontent.com/29051129/61011731-2a125100-a352-11e9-8a9f-14d4842e7742.png)

**Comment by flbulgarelli** [comment]
@luchotc @julian-berbel

**Comment by julian-berbel** [comment]
We just talked about it but I'm leaving this here too.

This is actually a bug in bash runner introduced (by me) here: mumuki/mumuki-bash-runner#21.

Since we were having non-deterministic behavior caused by the desynchronization of stdout and stderr I hacked it by redirecting stderr to stdout in every line. This however causes trouble with things like heredocs, which we are using in this particular exercise.
